src/evm_transition_tool/evmone.py

```diff
@@ -110,63 +119,52 @@ def evaluate(
         )
 
         if debug_output_path:
+            shutil.copytree(temp_dir.name, debug_output_path)
+            t8n_output_base_dir = os.path.join(debug_output_path, "t8n.sh.out")
+            t8n_call = " ".join(args)
+            for file_path in input_paths.values():  # update input paths
+                t8n_call = t8n_call.replace(
+                    os.path.dirname(file_path), os.path.join(debug_output_path, "input")
+                )
+            t8n_call = t8n_call.replace(  # use a new output path for basedir and outputs
+                temp_dir.name,
+                t8n_output_base_dir,
+            )
+            t8n_script = textwrap.dedent(
+                f"""\
+                #!/bin/bash
+                output_directory={os.path.join(t8n_output_base_dir, "output")}
```

Comment:
Similar comment to the last one.
Perhaps it would be safer to assume `t8n_output_base_dir` is the current working directory?

Reply:
As it stands:

https://github.com/ethereum/execution-spec-tests/blob/c7cc487cd3374c9222d8b64270773d370135abc4/src/evm_transition_tool/evmone.py#L137-L138

the `t8n.sh` script will delete the "output" sub-directory (where ever that might be), but doesn't seem to be dangerous. Only disadvantage is that files in the base directory aren't deleted, but I think we can live with that.